### ISSUE TITLE
Add browse/search groups users 

### ISSUE THREAD

**Issue by xavijam** [CartoDB/cartodb#5657]
<img width="1036" alt="screen shot 2015-09-22 at 12 11 44" src="https://cloud.githubusercontent.com/assets/132146/10015871/40056e18-6123-11e5-964d-935193e5107a.png">

@viddo

**Comment by viddo** [comment]
This was planned as a nice-to-have :point_right: #4883

**Comment by xavijam** [comment]
Ok! in that case, could you move to the next milestone or close if it is duplicated? porfaplease

**Comment by viddo** [comment]
It's planned TBD in this milestone, I'd added it to the list on the referenced issue for now

**Comment by viddo** [comment]
…planned to open issues as starting to work on those, so this is fine :)

**Comment by viddo** [comment]
More details, we have the same functionality and styles duplicated (with some minor variations):
![screen shot 2015-09-22 at 12 29 14](https://cloud.githubusercontent.com/assets/978461/10016217/933b5960-6125-11e5-87ba-c4172c3073f0.png)
IMHO it doesn't make sense to re-implement styles, we should be able to have components that can be used in various contexts without problem (in this case, in the settings page and in a modal)

**Comment by viddo** [comment]
@juanignaciosl follow-up: once #5781 is merged, the `PagedSearchView` and same approach used can be reused here to have the pagination and search functionality implemented. BUT that would require a GET endpoint for group users that doesn't exist yet - could you add one? (same approach as for the other collections)

**Comment by juanignaciosl** [comment]
I wouldn't add a new endpoint for that but make existing users endpoint to receive a `exclude_users_from_groups=1,2` parameter that exclude users from the groups you ask to, does it fit you use case?

Ok, but this won't be done today unless you tell me this is really urgent. If it is I'll do it right now. If it's not, open an issue, please.

**Comment by viddo** [comment]
No, it's not urgent, definitely don't have to be done (leaving for vacation tomorrow!).

Excluding users doesn't really fit, no, since the use-case is to search users by username/email (like the others I imagine, @saleiva can probably correct me here).

**Comment by juanignaciosl** [comment]
@viddo could you file an issue with the needs so I don't have to make guesses from the screenshots, please? We already have users endpoints, needing a new one is not probably going to be needed.

**Comment by viddo** [comment]
Frontend TBD
